### ISSUE TITLE
Something wrong with the "block" collision detection

### ISSUE THREAD

**Issue by smcameron** [smcameron/space-nerds-in-space#268]
I was playing around with the "BIGSHIP.LUA" script, and I noticed I can't fly my ship in between the various parts of the big ship like I am sure that I used to be able to do.  I end up colliding with something that isn't there that stops me.  Must have broken it somewhere along the way.

**Comment by smcameron** [comment]
Git bisect says it's this commit: cc6dd73144ba95c093dc96cc46cfa2d359c122ab

```
scameron@wombat ~/github/snis-bisect/space-nerds-in-space $ git bisect bad
cc6dd73144ba95c093dc96cc46cfa2d359c122ab is the first bad commit
commit cc6dd73144ba95c093dc96cc46cfa2d359c122ab
Author: Stephen M. Cameron <stephenmcameron@gmail.com>
Date:   Mon Mar 4 09:58:37 2019 -0500

    Make block type object use shape_collision
    
    Signed-off-by: Stephen M. Cameron <stephenmcameron@gmail.com>

:100644 100644 08adac262420ce1a05e4c527fccdce2dd1973697 318b529ecb889492b78b0aa61baa56356efdcaec M	snis.h
:100644 100644 0cc9e535f5c16dd7cda08a0d5dac0f3c4274c9af d24c32a60b97fd6ea2fac1013435ecf573f14eee M	snis_client.c
:100644 100644 fc2be219f7511b57a133c76e1a6885535cce17eb 05fb3779e40f95c481645fec0ba8f1f351b5f5f3 M	snis_debug.c
:100644 100644 300fdfebd13ade23241a60383fecadbabfe60d02 8943f59ce2982ac48d36380056efbec668785f9a M	snis_server.c
```

**Comment by smcameron** [comment]
Testing with share/snis/luascripts/TEST/TESTSPHEROID.LUA, the problem appears to be with SHAPE_CAPSULE.  SHAPE_CUBOID and I think SHAPE_SPHERE is probably fine too.  I am suspicious I may have done something like use a diameter as a radius.

**Comment by smcameron** [comment]
Ok, I think the collision detection appears to be correct, I think it's the drawing that's not correct.  The cylinder model share/snis/models/cylinder.scad has a radius of 0.5 rather than 1.0.

**Comment by smcameron** [comment]
Fixed by c9a1f11a2a8edbb07f9453f1cd7f10615cc86dde  Fix drawing of sphere and capsule shapes.
